code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm

```diff
@@ -491,4 +491,75 @@ Difficulty: Medium
 	butcher_results = list(/obj/item/stack/ore/diamond = 5, /obj/item/stack/sheet/sinew = 5, /obj/item/stack/sheet/bone = 30)
 
 /mob/living/simple_animal/hostile/megafauna/dragon/lesser/grant_achievement(medaltype,scoretype)
-	return
\ No newline at end of file
+	return
+	
+/mob/living/simple_animal/hostile/megafauna/dragon/space_dragon
+	name = "space dragon"
+	maxHealth = 250
+	health = 250
+	faction = list("neutral")
+	desc = "A space carp turned dragon by vile magic.  Has the same ferocity of a space carp, but also a much more enabling body."
+	icon = 'icons/mob/spacedragon.dmi'
+	icon_state = "spacedragon"
+	icon_living = "spacedragon"
+	icon_dead = "spacedragon_dead"
+	obj_damage = 80
+	melee_damage_upper = 35
+	melee_damage_lower = 35
+	speed = 0
+	mouse_opacity = MOUSE_OPACITY_ICON
+	loot = list()
+	crusher_loot = list()
+	butcher_results = list(/obj/item/stack/ore/diamond = 5, /obj/item/stack/sheet/sinew = 5, /obj/item/stack/sheet/bone = 30)
+	move_force = MOVE_FORCE_NORMAL
+	move_resist = MOVE_FORCE_NORMAL
+	pull_force = MOVE_FORCE_NORMAL
+	deathmessage = "screeches as its wings turn to dust and it collapses on the floor, life estinguished."
+	var/datum/action/small_sprite/carpsprite = new/datum/action/small_sprite/spacedragon()
+
+/mob/living/simple_animal/hostile/megafauna/dragon/space_dragon/grant_achievement(medaltype,scoretype)
+	return	
+	
+/mob/living/simple_animal/hostile/megafauna/dragon/space_dragon/Initialize()
+	carpsprite.Grant(src)
+	mob_spell_list += new /obj/effect/proc_holder/spell/aoe_turf/repulse/spacedragon(src)
+	. = ..()
+	smallsprite.Remove(src)
+
+/mob/living/simple_animal/hostile/megafauna/dragon/space_dragon/proc/fire_stream(var/atom/at = target)
```

Comment:
does this work (argument)? I didn't know the target var was useful for PCs

Reply:
It does.

Reply:
The ash drake uses the same argument as well.

Reply:
nice